README.md

```diff
@@ -112,7 +112,7 @@ Using the SlackService
 ```typescript
 import { Controller } from '@nestjs/common';
 import { ConfigService } from '@nestjs/config';
-import { SlackService } from 'nestjs-slack-bolt/services/slack.service';
+import { SlackService } from 'nestjs-slack-bolt/dist/services/slack.service';
```

Comment:
this is incorrect. you don't need to add /dist when using this package. the entrypoint already exports everything from the dist directory.